vnext/ReactUWP/Modules/Animated/ValueAnimatedNode.cpp

```diff
@@ -40,7 +40,10 @@ double ValueAnimatedNode::RawValue() {
 }
 
 void ValueAnimatedNode::RawValue(double value) {
-  m_propertySet.InsertScalar(s_valueName, static_cast<float>(value));
+  if (RawValue() != value) {
```

Comment:
>(RawValue() != value [](start = 5, length = 20)

For double, maybe it's not the best way to do the comparison. I don't know if you need to switch to [https://bitbashing.io/comparing-floats.html](https://bitbashing.io/comparing-floats.html)